src/app_charts/base/robot/gcr-credential-refresher.yaml

```diff
@@ -17,6 +17,7 @@ spec:
       - image: {{ .Values.registry }}{{ .Values.images.gcr_credential_refresher }}
         args:
         - --robot_id_file=/credentials/robot-id.json
+        - --sa_name={{ .Values.robot.defaultSAName }}
```

Comment:
Let's make this `service_account_name` so it's painfully obvious.

the SA shorthand shows up a couple times in code as well. I think there' it's fine. But here it's better to tell the reader everything at a glance.